cumulusci/tasks/push/tasks.py

```diff
@@ -231,13 +237,31 @@ def _init_options(self, kwargs):
 
         # Set the namespace option to the value from cumulusci.yml if not
         # already set
+        if "orgs" not in self.options and "csv" not in self.options:
```

Comment:
Seems like you need _either_ `orgs` or `csv`, but not _both_. 

I would expect that, if the user passes both, you'd raise an exception.